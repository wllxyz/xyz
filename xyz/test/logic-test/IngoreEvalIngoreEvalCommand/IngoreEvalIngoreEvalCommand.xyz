<i>-->""==><o>-->($IGNORE, ($EVAL, ($IGNORE, ($EVAL, ($ADD, "0", "1") ) ) ) );
