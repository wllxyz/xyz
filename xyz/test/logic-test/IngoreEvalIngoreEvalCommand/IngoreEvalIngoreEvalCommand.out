$LEFT_QUOTE$EVAL$SEPERATOR$LEFT_QUOTE$EVAL$SEPERATOR1$RIGHT_QUOTE$RIGHT_QUOTE